pom.xml

```diff
@@ -556,6 +556,16 @@
             </properties>
         </profile>
         <profile>
+            <id>spark_2.1.1</id>
+            <properties>
+                <spark-version.project>2.0</spark-version.project>
```

Comment:
By the way, this profile isn't really needed. The profile only really has to exist to select the 2.0 vs 1.5-plus module. from there callers can easily modify `-Dspark.version=...` as needed. I get that it's a convenience, but also a maintenance burden. For example, you don't want a cloudera-2.1 profile; you just need to use `-Pcloudera` to activate the repo and then select `-Dspark.version=` accordingly.

Of course this doesn't solve the bigger issue of how many different versions of the library to publish, which is a real problem. I'm slugging through that now; it's going to be tricky, but might be able to make one Spark 2 binary work for everything.